models/image.py

```diff
@@ -1728,6 +1735,17 @@ def get_downstreams(self, session=None):
 
             return downstreams
 
+    def get_psf(self):
+        """Load the PSF object for this image.
+
+        If it is a sub image, it will load the PSF from the new image.
```

Comment:
Note that this does hardcode the assumption that it's only the reference image we're convolving if the sub doesn't have its own PSF.  Generally that's what I do in alard-lupton subtractions, but I don't know if we want to bake that in.

Reply:
The point of `get_psf` is to make the best effort to get the most relevant PSF. 
Since the ZOGY subtraction gives us a PSF image but without the bells and whistles a full PSF object, we are forced to go back to the new image for the next best thing. 
I have an issue to improve and unify the ZOGY output with our PSF model (#194).

Reply:
But isn't the new psf wrong for the ZOGY subtraction?  I thought in ZOGY that both images got convolved with something.

Reply:
If the reference has a narrower PSF than the new image, then the ZOGY PSF will look a lot like the new PSF. 

The right thing to do is to always use the PSF from ZOGY (when using that subtraction) but to store it in a way that is compatible with the PSF model, and save it to DB with its own provenance and so on. 

As far as I can tell the PSF photometry in Measurements is the only place where that would be used... and I am not sure how good PSF photometry is going to turn out to be (in the end, everyone just uses aperture photometry).